functional/main_test.py

```diff
@@ -171,10 +171,8 @@ def test_register_link_unauthorized(self):
     def test_order_subscription_link(self):
```

Comment:
Why was this test changed?
What does it test now?

Reply:
Commited hunk by mistake, fixed